cron/getIncrementBalance.js

```diff
@@ -0,0 +1,46 @@
+const cron = require('node-cron');
+const { familyModel, userModel, transactionModel } = require('../database/models');
+
+const { getLogger } = require('../helpers');
+const logger = getLogger('getIncrementBalance');
+
+async function main() {
+  try {
+    await cron.schedule('0 0 1 * *', async function () {
+
+      const allFamilies = await familyModel.find({});
+
+      await Promise.all(allFamilies.map(async item => {
+        await item.update(
+          {
+            $inc: {
+              giftsForUnpacking: Math.floor((item.balance * item.flatSquareMeters / item.flatPrice) - item.giftsUnpacked)
+            }
+          }
+        )
+
+        const user = await userModel.findOne({ familyId: item.familyId });
```

Comment:
Imagine that we have multiple users for family. Then this logic would simply assign income transaction to random user. Probably better to make `userId` in transaction optional